src/libraries/System.Security.Cryptography.Cng/src/Internal/Cryptography/BasicSymmetricCipherNCrypt.cs

```diff
@@ -116,11 +116,11 @@ private void Reset()
             if (IV != null)
             {
                 CngProperty prop = new CngProperty(Interop.NCrypt.NCRYPT_INITIALIZATION_VECTOR, IV, CngPropertyOptions.None);
-                _cngKey.SetProperty(prop);
+                _cngKey!.SetProperty(prop);
             }
         }
 
-        private CngKey _cngKey;
+        private CngKey? _cngKey;
```

Comment:
seems `_cngKey` is set in constructor and better to be non null, only nulled in `Dispose()`